core-service/src/cmd/server/handler.go

```diff
@@ -55,8 +55,9 @@ func NewHandler(cfg *config.Config, apm *utils.Apm, u *Usecases, logger utils.Lo
 
 	e := echo.New()
 
-	e.HidePort = true // to do: make this true on production env
-	e.HideBanner = true
+	isProd := config.LoadAppConfig().Env == "production"
```

Comment:
```suggestion
	isProd := cfg.Env == "production"
```
maybe can use env from cfg 😄

Reply:
sure nice, thanks here what you want [use config from main args](https://github.com/jabardigitalservice/portal-jabar-services/pull/353/commits/57d804de44a079bbfbbdfacdbd1cf14498e7dc10) 😄